fhir-search/src/main/java/com/ibm/fhir/search/location/uom/standard/StatuteLengthUnits.java

```diff
@@ -46,7 +50,11 @@
 
     MILE("mi_us", 8 * 40 * 16.5),
     MILE_CI("MI_US", 8 * 40 * 16.5),
-    
+    // Support for common variants that are not in UCUM
+    MILE_VARIANT_MIS("mis", 8 * 40 * 16.5),
+    MILE_VARIANT_MILE("mile", 8 * 40 * 16.5),
+    MILE_VARIANT_MILES("miles", 8 * 40 * 16.5),
```

Comment:
no `mi` ?

Reply:
thanks, it's hard to see the tree in the forest